connect-inject/handler.go

```diff
@@ -345,7 +345,25 @@ func (h *Handler) Mutate(req *v1beta1.AdmissionRequest) *v1beta1.AdmissionRespon
 	// Add annotations so that we know we're injected
 	patches = append(patches, updateAnnotation(
 		pod.Annotations,
-		map[string]string{annotationStatus: "injected"})...)
+		map[string]string{
+			annotationStatus: "injected",
+		})...)
+
+	// Consul-ENT support
+	if h.EnableNamespaces && h.EnableK8SNSMirroring {
+		patches = append(patches, updateAnnotation(
+			pod.Annotations,
+			map[string]string{
+				annotationConsulDestinationNamespace: h.consulNamespace(req.Namespace),
+			})...)
+	}
```

Comment:
Ah yeah I missed this in the last round of rip-outs.. it will go away in this PR and unit tests are covered when ent support comes in.

We have already unit tests for the patch framework for the labels though.

Reply:
yeah..this was on an older commit